### PR TITLE
[WIP] Add git-ssh-test

### PR DESCRIPTION
**Pull request by praveen4g0**
* Now we add git-ssh tests to authenticate git tasks (pull & push), tagged with `ssh`

### CODE REVIEW

**Comment by praveen4g0** [comment]
cc: @ppitonak 
please do review and merge this pr :)

File: specs/pipelines/run.spec [hunk 1]
```diff
@@ -102,3 +102,16 @@ Steps:
...
       |1   |pipelinerun-with-pipelinespec-taskspec-vb|successfull|no                     |
+
+## Use SSH authentication in `git` type `tasks`(without know_hosts)
+Tags: e2e, integration, pipelines, ssh
+
+Steps:
+  * Create
+      |S.NO|resource_dir                             |
+      |----|-----------------------------------------|
+      |1   |testdata/v1beta1/pipelinerun/git-ssh.yaml|
```
Comment (ppitonak) [inline specs/pipelines/run.spec:113]: why does the path contain `v1beta1`?

File: testdata/v1beta1/pipelinerun/git-ssh.yaml [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+        #!/usr/bin/env sh
+        echo "Running: '$(params.commands)'"
+        $(params.commands)
+  workspaces:
+    - name: source
+---
+apiVersion: tekton.dev/v1beta1
+kind: Pipeline
+metadata:
+  name: welcome-app-pipeline
```
Comment (ppitonak) [inline testdata/v1beta1/pipelinerun/git-ssh.yaml:41]: Why `welcome-app`? I would understand something with `git-ssh` or `vote-api`

**Comment by ppitonak** [comment]
Outdated

**Pull request closed without merging**